### ISSUE TITLE
Header Responsiveness on Small Screens

### ISSUE THREAD

**Issue by Offroaders123** [photopea/photopea#2736]
I tried using the site on a mobile device to see if the interfaces were still responsive to smaller displays, and it worked very well! I only found an issue with the header (menubar/tool option sections) when it was wider than the viewport. When this is the case, it would allow the user to scroll left and right to access the other options that weren’t accessible on screen. While the header can scroll left and right, the extra overflow somehow expands beyond the width of the rest of the site, causing the body to have horizontal overflow.

To test this, I recommend going to the site on an iPhone or Android phone, as the viewport has to be smaller than the total width of the header elements. Opening the site with DevTools responsive mode didn’t seem to show the issue because it didn’t cause the unintended horizontal overflow, so it may only be present on mobile devices for some reason. I didn’t get to try this on an Android device, but I imagine that it will be present there as well.

**Comment by photopea** [comment]
Hi, when your screen is not wide enough, you should be able to click and drag ("move") the top menu, to get access to hidden items.

Have you tried it? I do not understand what you mean by "overflow".

**Comment by Offroaders123** [comment]
https://user-images.githubusercontent.com/65947371/103988633-51163280-5143-11eb-9696-194ba1f176ad.mov

Sorry for the confusion, I added a video that should help convey the issue better.

**Comment by photopea** [comment]
So you mean the fact, that the rest of the editor is scrolling, not just the top menu? This does not happen in Chrome for Android.

I think it could be a bug in iOS Safari. Is your iOS updated? If not, could you update it?

**Comment by Offroaders123** [comment]
Yes, it appears that just the header should be able to scroll, but the body should not be scrollable/resized to the size of the header.
I updated to the latest release of `iOS 14.3`, and the issue is still present.

I encountered the same issue on Safari while developing my site as well, and this is the way that I solved what appeared to be the same issue, by styling the root HTML tag with the following style properties:
<pre>
html {
  position: fixed;
  width: 100%;
}

  /*
     The width property would have to be 100% rather than 100vw because Safari
     tends to treat *vw units differently than other browsers depending on the
     document size, from my own experiments and testing.
  */
</pre>
I wasn't able to try these attributes on the active `photopea.com` to see if it makes a difference, but I'm fairly sure it may be a possible solution since it worked for my site and a few others I found on Stack Overflow.

I also made an animated SVG image to show what the header is doing to the size of the body. The blue line (outline of the body element) from the image should be underneath the green line (outline of the viewport) during the scroll process, and the image animation shows what is happening instead of this.
[Photopea Header Issue.zip](https://github.com/photopea/photopea/files/5787036/Photopea.Header.Issue.zip)
